bears/c_languages/CPPCleanBear.py

```diff
@@ -22,5 +23,14 @@ class CPPCleanBear:
     CAN_DETECT = {'Smell', 'Unused Code', 'Security'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-        return filename,
+    def create_arguments(filename,
+                         file,
+                         config_file,
+                         include_paths: typed_list(str)=()):
+        args = []
+        args.append(filename)
+        for part in include_paths:
```

Comment:
just call this variable `path` :)
(or `include_path`. Some optional opinion from me: I usually try to name variables, that contain lists, tuples, sequences with a plural-"s", e.g. `things = []` and not `thing = []`. When I iterate over them, I just leave the `s` for the iteration variable: `for thing in things`. Just some inspirations ;D)

Reply:
Thank you so much @Makman2 yeah I do realize that improves code readability by a great extent :)